b2sdk/transfer/emerge/executor.py

```diff
@@ -278,8 +311,17 @@ def _find_unfinished_file_by_plan_id(
         ):
             if file_.file_info != file_info:
                 continue
+            # FIXME: encryption is None ???
             if encryption is None or file_.encryption != encryption:
                 continue
+            if bool(legal_hold) != file_.legal_hold:
```

Comment:
this series of ifs is very similiar to the one in `_match_unfinished_file_if_possible` - shouldn't this be moved to another (small) method?